### PR TITLE
Json conversion

### PR DESCRIPTION
**Pull request by jd0yle**

### CODE REVIEW

File: rules-topics/18.00_glossary.md [hunk 1]
```diff
@@ -1,84 +1,201 @@
...
+| 4 | Spell Parry |
+| 4 | Weakening Strike |
+| 5 | Breaching Strike |
+| 5 | Silencing Strike |
+| 6 | Fearful Strike |
+| 6 | Waylay |
+| 7 | Critical Empowerment |
+| 7 | Warbow Training |
+| 8 | Stunning Strike  |
+| 8 | Trickle Blade |
```
Comment (sd0yle) [inline rules-topics/18.00_glossary.md:24]: Surprising Proficiency is missing here.

File: data/agility-abilities.json [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+    },
+    {
+        "name": "Stunning Strike ",
+        "pool": "Agility",
+        "level": "8",
+        "duration": "20 Seconds",
+        "verbal": "Physical Stun",
+        "description": "This skill is a strike that renders the target stunned for 20 seconds.  Stunned target may not move, communicate in any manner, activate or use items, or use in-game skills.  This effect may be affected by the Transfer Malady or Cure Body. The effect of this skill may not be stacked or extended. This is a STUN effect."
+    },
+    {
```
Comment (sd0yle) [inline data/agility-abilities.json:138]: Surprising Proficiency is missing here.

File: rules-topics/09.00_agility-abilities.md [hunk 3]
```diff
@@ -9,254 +9,187 @@ The Agility Point cost of each ability is equal to its level.  Characters may "m
...
+| 6 | Fearful Strike |
 | 6 | Waylay |
-| 6 | Fear |
-| 7 | Varible Critical |
+| 7 | Critical Empowerment |
 | 7 | Warbow Training |
-| 8 | Stun |
-| 8 | Toxin Strike |
+| 8 | Stunning Strike  |
+| 8 | Trickle Blade |
```
Comment (sd0yle) [inline rules-topics/09.00_agility-abilities.md:28]: Surprising Proficiency is missing here.

File: rules-topics/09.00_agility-abilities.md [hunk 4]
```diff
@@ -9,254 +9,187 @@ The Agility Point cost of each ability is equal to its level.  Characters may "m
...
-| Ability | **Stun**   |
+| | **Stunning Strike ** |
 |:---|---|
-| **Type**| Agility |
-| **Level**| 8 |
+| **Level** | 8 |
 | **Duration** | 20 Seconds |
-| **Call**| "Physical Stun"  |
+| **Verbal** | "Physical Stun" |
 | **Description** | This skill is a strike that renders the target stunned for 20 seconds.  Stunned target may not move, communicate in any manner, activate or use items, or use in-game skills.  This effect may be affected by the Transfer Malady or Cure Body. The effect of this skill may not be stacked or extended. This is a STUN effect. |
```
Comment (sd0yle) [inline rules-topics/09.00_agility-abilities.md:154]: Surprising Proficiency is missing here.

**Review by sd0yle — CHANGES_REQUESTED** [review]

File: rules-topics/18.00_glossary.md (hunk 1 above)
Comment (jd0yle) [reply rules-topics/18.00_glossary.md:24]: Only needs to be added to the agility json

File: rules-topics/09.00_agility-abilities.md (hunk 3 above)
Comment (jd0yle) [reply rules-topics/09.00_agility-abilities.md:28]: Only needs to be added to the agility json

File: rules-topics/09.00_agility-abilities.md (hunk 4 above)
Comment (jd0yle) [reply rules-topics/09.00_agility-abilities.md:154]: Only needs to be added to the agility json

**Review by sd0yle — DISMISSED** [review]

**Review by sd0yle — APPROVED** [review]

**Pull request merged**